### PR TITLE
update branding

### PR DESCRIPTION
**Pull request by zbtirrell**

### CODE REVIEW

**Review by sc0ttkclark — APPROVED** [review]

File: image-widget.php [hunk 1]
```diff
@@ -573,8 +573,8 @@ private function upgrade_nag_plugins_admin_msg() {
 
 		$msg = sprintf(
 			__( '<p class="dashicons-before dashicons-format-gallery"><strong>Image Widget Plus</strong> - Add lightbox, slideshow, and random image widgets. <strong><a href="%s" target="_blank">Find out how!</a></strong></p>', 'image-widget' ),
-			'http://m.tri.be/19my',
-			'http://m.tri.be/19my'
+			'https://evnt.is/19my',
+			'https://evnt.is/19my'
```
Comment (Camwyn) [inline image-widget.php:577]: redundant second url?

Comment (sc0ttkclark) [reply image-widget.php:577]: Indeed, there's only one placeholder.

```suggestion
			'https://evnt.is/19my'
```

Comment (sc0ttkclark) [inline image-widget.php:577]:
```suggestion
			'https://evnt.is/19my'
```

**Review by sc0ttkclark — APPROVED** [review]

**Comment by sc0ttkclark** [comment]
@bordoni needs SVN commit

**Comment by sc0ttkclark** [comment]
Released to SVN

**Pull request merged**